platform-tests/src/acceptance/rds_broker_test.go

```diff
@@ -105,6 +106,93 @@ var _ = Describe("RDS broker", func() {
+		Expect(plans.Out.Contents()).To(ContainSubstring("S-dedicated-5.7"))
+		Expect(plans.Out.Contents()).To(ContainSubstring("S-HA-dedicated-5.7"))
+		Expect(plans.Out.Contents()).To(ContainSubstring("M-dedicated-5.7"))
+		Expect(plans.Out.Contents()).To(ContainSubstring("M-HA-dedicated-5.7"))
+		Expect(plans.Out.Contents()).To(ContainSubstring("L-dedicated-5.7"))
+		Expect(plans.Out.Contents()).To(ContainSubstring("L-HA-dedicated-5.7"))
+	})
+
+	Context("creating a database instance", func() {
+		// Avoid creating additional tests in this block because this setup and teardown is
+		// slow (several minutes).
+
+		var (
+			appName         string
+			dbInstanceName  string
+			rdsInstanceName string
+		)
+		BeforeEach(func() {
+			appName = generator.PrefixedRandomName("CATS-APP-")
+			dbInstanceName = generator.PrefixedRandomName("test-db-")
+			Expect(cf.Cf("create-service", serviceName, testPlanName, dbInstanceName).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+
+			pollForRDSCreationCompletion(dbInstanceName)
+
+			rdsInstanceName = getRDSInstanceName(dbInstanceName)
+			fmt.Fprintf(GinkgoWriter, "Created RDS instance: %s\n", rdsInstanceName)
+
+			Expect(cf.Cf(
+				"push", appName,
+				"--no-start",
+				"-b", config.GoBuildpackName,
+				"-p", "../../example-apps/healthcheck",
+				"-f", "../../example-apps/healthcheck/manifest.yml",
+				"-d", config.AppsDomain,
+			).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+
+			Expect(cf.Cf("bind-service", appName, dbInstanceName).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+
+			Expect(cf.Cf("start", appName).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+		})
+
+		AfterEach(func() {
+			cf.Cf("delete", appName, "-f").Wait(DEFAULT_TIMEOUT)
+
+			Expect(cf.Cf("delete-service", dbInstanceName, "-f").Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+
+			// Poll until destruction is complete, otherwise the org cleanup (in AfterSuite) fails.
+			pollForRDSDeletionCompletion(dbInstanceName)
+		})
+
+		It("binds a DB instance to the Healthcheck app that matches our criteria", func() {
+			By("allowing connections from the Healthcheck app")
+			resp, err := httpClient.Get(helpers.AppUri(appName, "/db?service=mysql"))
+			Expect(err).NotTo(HaveOccurred())
+			body, err := ioutil.ReadAll(resp.Body)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(200), "Got %d response from healthcheck app. Response body:\n%s\n", resp.StatusCode, string(body))
+
+			By("disallowing connections from the Healthcheck app without TLS")
+			resp, err = httpClient.Get(helpers.AppUri(appName, "/db?service=mysql&ssl=false"))
+			Expect(err).NotTo(HaveOccurred())
+			body, err = ioutil.ReadAll(resp.Body)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.StatusCode).NotTo(Equal(200), "Got %d response from healthcheck app. Response body:\n%s\n", resp.StatusCode, string(body))
+			Expect(body).To(MatchRegexp("no pg_hba.conf entry for .* SSL off"), "Connection without TLS did not report a TLS error")
```

Comment:
This doesn't look right. It looks like a Postgres specific error message that I wouldn't expect from a MySQL instance.